dockerbuild/config/config.php

```diff
@@ -8,8 +8,9 @@
 
 use Sil\PhpEnv\Env;
 use Sil\PhpEnv\EnvVarNotFoundException;
+use SimpleSAML\Utils;
 
-$httpUtils = new \SimpleSAML\Utils\HTTP();
+$httpUtils = new Utils\HTTP();
```

Comment:
I was expecting the use to include the HTTP, but I guess if the other two have HTTP defined, this would make sense.

Reply:
I followed the example that SimpleSAMLphp devs use. Plus, I like how it reads: `new Utils\HTTP` rather than `new HTTP`.